cmd/probe.go

```diff
@@ -13,8 +13,14 @@ var probeInstallOptions probe.Options
 // probeCmd represents the get command
 var probeCmd = &cobra.Command{
 	Use:   "probe",
-	Short: "Display probe information",
-	Long:  `Display probe information`,
+	Short: "Checks for supported KubeArmor features in the current environment",
+	Long: `Checks for supported KubeArmor features in the current environment.
+
+If KubeArmor is not running, it does a precheck to know if kubearmor will be supported in the environment
+and what KubeArmor features will be supported e.g: observability, audit, etc. 
```

Comment:
```suggestion
and what KubeArmor features will be supported e.g: observability, enforcement, etc. 
```
Since enforcement is an highlight :)